administrator/components/com_admin/sql/updates/sqlazure/3.7.0-2016-09-01.sql

```diff
@@ -0,0 +1,9 @@
+SET IDENTITY_INSERT [#__extensions] ON;
+
+INSERT INTO [#__extensions] ([extension_id], [name], [type], [element], [folder], [client_id], [enabled], [access], [protected], [manifest_cache], [params], [custom_data], [system_data], [checked_out], [checked_out_time], [ordering], [state])
+SELECT 33, 'com_associations', 'component', 'com_associations', '', 1, 1, 1, 1, '', '', '', '', 0, '1900-01-01 00:00:00', 0, 0;
+
+SET IDENTITY_INSERT [#__extensions] OFF;
+
+INSERT INTO [#__menu] ([menutype], [title], [alias], [note], [path], [link], [type], [published], [parent_id], [level], [component_id], [checked_out], [checked_out_time], [browserNav], [access], [img], [template_style_id], [params], [lft], [rgt], [home], [language], [client_id])
+SELECT 'menu', 'com_associations', 'Multilingual Associations', '', 'Multilingual Associations', 'index.php?option=com_associations', 'component', 1, 1, 1, 33, 0, '1900-01-01 00:00:00', 0, 1, 'class:associations', 0, '', 43, 44, 0, '*', 1
```

Comment:
look at last character
also you need to start/end SET IDENTITY_INSERT [#__menu] ON/OFF;